Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -35,10 +35,8 @@ There are usually two release engineers working at any given time. It goes witho
 ### Phase 2: Initiation
 *It starts at the end of the sprint (typically when the new sprint starts on Monday)*
 
-1. Cut a release branch, naming it using the `release/{appname}/{version}` convention (with `{appname}` being one of `babylon`/`telus`/`bupa`/`nhs111`)
-   * If you are releasing the main Babylon app, create the new branch from `develop`, and name it `release/babylon/{version}` (e.g. `release/babylon/4.1.0`)
-   * If you're releasing another app (e.g. Telus, Bupa, NHS111), since they typically go thru the release process _only_ once the main Babylon app has been signed off by QA, you should create the new branch from the corresponding Babylon release branch that was recently already QA'd and signed off (e.g. `release/babylon/4.1.0`) instead of `develop`, and name your new branch using the same `release/{appname}/{version}` convention (e.g. `release/telus/4.1.0`)
-   * Push the branch to `origin`
```

Comment:
if we do not do automatic cut-offs for other targets then this step should not be removed

Reply:
🤦‍♂

Reply:
please use github suggestions @viorel15